ebos/ecloutputblackoilmodule.hh

```diff
@@ -108,6 +114,70 @@ class EclOutputBlackOilModule
         };
         static const int numFipValues = PoreVolume + 1 ;
     };
+    struct WellProdDataType
+    {
+        enum WPId
+        {
+            WellLocationi = 0, //WLi
```

Comment:
At first sight this seems like a duplication of code/functionality which is already in opm-common?

Reply:
I am not sure of what is in opm-common.